iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/KubernetesParserTest.java

```diff
@@ -272,6 +250,175 @@ void shouldRemoveLineNumberCommentForEndDocumentAfterEvaluation() throws IOExcep
       "metadata: #8",
       "spec: #9",
       "... #10");
+
+    FileTree file = parseTemplate("{{ dummy helm }}", evaluated);
+
+    assertThat(file.documents().get(0).children()).hasSize(4);
+    assertThat(file.template()).isEqualTo(FileTree.Template.HELM);
+  }
+
+  @Test
+  void shouldFindShiftedLocation() throws IOException, URISyntaxException {
```

Comment:
I believe simply copying these tests from `CommentLocationVisitorTest` here adds a bit of an overhead, because in these tests we only check pre-processing part (marking lines) and shifting, but execute everything including YAML parsing. WDYT about moving this logic e.g into `HelmProcessor` (maybe as a separate ticket)?

Reply:
I agree it's a bit of overhead, but I didn't find a better place for those tests.
The original `CommentLocationVisitorTest` also executed YAML parsing. In both cases, the Go executable is mocked.